WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.kt

```diff
@@ -1967,20 +1967,25 @@ class EditPostActivity : BaseAppCompatActivity(), EditorFragmentActivity, Editor
      * 2. Saves the post via [EditPostActivity.updateAndSavePostAsync];
      * 3. Invokes the listener method parameter
      */
-    private fun updateAndSavePostAsyncOnEditorExit(listener: OnPostUpdatedFromUIListener?) {
+    private fun updateAndSavePostAsyncOnEditorExit(listener: OnPostUpdatedFromUIListener?,
+                                                   isFinishing: Boolean = false) {
```

Comment:
```suggestion
    private fun updateAndSavePostAsyncOnEditorExit(
        listener: OnPostUpdatedFromUIListener?,
        isFinishing: Boolean = false
    ) {
```

Just following a more Kotlint style